app/services/preserved_object_handler.rb

```diff
@@ -270,9 +270,6 @@ def update_pc_invalid_moab
   # given a PreservedCopy instance and whether the caller found the expected version of it on disk, this will perform
   # other validations of what's on disk, and will update the status accordingly
   def set_status_as_seen_on_disk(pres_copy, found_expected_version)
-    # TODO: when we implement C2M, we should add a check here to look for the object on disk, and set PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS
```

Comment:
yay!  removal of a TODO with no action required!

Reply:
uh oh - I forgot you did this here -- if #544 gets merged first there will be a conflict here.  Sorry!

Reply:
git dealt with this with no human intervention (did the whole rebase of this branch with no human intervention after #544 was merged to master).  not that it'd have been a terrible conflict anyway.